CONTRIBUTING.md

```diff
@@ -21,7 +24,22 @@ On <https://github.com/OpenMined/PySyft/issues> you can find all open Issues. Yo
 To contribute to PySyft you will need to fork the OpenMind/PySyft repository.
 Then you can work risk-free on your fork.
 
-You will just need to fork once. After that you can call `git fetch upstream` and `git pull 'branch-name'` before you do your local changes to get the remote changes and be up-to-date
+- Fork this Repository. This will create a local copy (Remote Repository) of this repository to your Github profile. Keep a reference to the original project in `upstream` remote.
+```
+$ git clone https://github.com/<your-username>/<repo-name>
+$ cd <repo-name>
+$ git remote add upstream https://github.com/<upstream-owner>/<repo-name>
+```
+
+### Syncing a Forked Repository
+
+To sync your fork (remote) with the OpenMined/PySyft (upstream) repository please see this [Guide](https://help.github.com/articles/syncing-a-fork/) on how to sync your fork or follow the given commands.
+
+```
+$ git remote update
+$ git checkout <branch-name>
+$ git rebase upstream/<branch-name>
```

Comment:
Q: I think here you also need to push? to make the changes appear in the upstream? (push into your local fork)

Reply:
@gmuraru yes, here I have written the commands only to update the local repository in their machine, so after the changes are made, they will push it to the repository. to avoid confusion I write it down here,